packages/autocomplete-core/src/utils/isAndroid.ts

```diff
@@ -0,0 +1,5 @@
+const regex = /android[-/ ]?([\w.]*)/i;
+
+export function isAndroid(userAgent: string) {
+  return Boolean(userAgent?.match(regex));
```

Comment:
Why `?` here? (same below)

```suggestion
  return Boolean(userAgent.match(regex));
```

Reply:
The user agent could be `undefined`. In the `metadata.test.ts`, `window` is not defined, so the test fails.
Should we check for the user agent and return `false` earlier instead?

Reply:
In your function signature, `userAgent` is always defined so the check either needs to happen earlier or the signature needs to change.